.github/workflows/services-json.yml

```diff
@@ -0,0 +1,36 @@
+name: Services Validator
+
+on:
+  push:
```

Comment:
The push rule is missing the path filter, I guess this was temporary for testing? Otherwise it should use the same path filter. Also since the annotation doesn't happen for non-PRs how is the failure logged?

Reply:
If any validation errors are found and it's not a PR, it'll be treated properly as an error in the 'Validate Service JSON Schema' step itself, as the script will return a status code of 1 via `sys.exit(1)`.

Reply:
Does the action support adding annotations to files in a commit rather than just leaving the errors in this log?

Reply:
I am actually unsure. It's likely. I'll test it and get back to you.

Reply:
I have confirmed that annotations do work on straight commits, so we could enable the Annotate Errors step on all runs.

![image](https://user-images.githubusercontent.com/941350/164885587-be2303e5-4448-4591-a268-c06007d43994.png)

Reply:
Yeah I also think it might be better to have CI just check for errors but committers should run the validator locally before committing anyway, so a local check should then give the explicit error anyway.